modules/flowable-cmmn-converter/src/test/java/org/flowable/test/cmmn/converter/CasePageTaskCmmnXmlConverterTest.java

```diff
@@ -49,64 +48,66 @@ public void convertModelToXML() throws Exception {
     }
     
     public void validateModel(CmmnModel cmmnModel) {
-        assertNotNull(cmmnModel);
-        assertEquals(1, cmmnModel.getCases().size());
+        assertThat(cmmnModel).isNotNull();
         
         // Case
-        Case caze = cmmnModel.getCases().get(0);
-        assertEquals("casePageCase", caze.getId());
+        assertThat(cmmnModel.getCases())
+            .isNotNull()
+            .hasSize(1)
+            .extracting(Case::getId)
+            .containsExactly("casePageCase");
```

Comment:
Just FYI when using `containsExactly` there is no need for `isNotNull` and `hasSize`. Since `containsExactly` will fail if the element is null (it does not contain exactly) and if the size differs (it does not contain exactly). There are other places where the same construct is used.

Reply:
Done; went through all files in package and made the same change.  Also reformatted to new style.